src/sonic-config-engine/minigraph.py

```diff
@@ -285,18 +285,33 @@ def get_mgmt_info(devices, dev, port):
 
     return ret_val
 
-def get_alias_map_list(hwsku):
-    alias_map_json = os.path.join('/usr/share/sonic', hwsku, 'alias_map.json')
-    if not os.path.isfile(alias_map_json):
+def get_alias_map_list(hwsku, platform=None):
+    port_config_candidates = []
+    port_config_candidates.append('/usr/share/sonic/sku/port_config.ini')
```

Comment:
Don't get your question.

---
In reply to: [101696721](https://github.com/Azure/sonic-buildimage/pull/304#discussion_r101696721) [](ancestors = 101696721)